mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
```

Comment:
Inconsistent naming convention. The function `eraseOrder` uses camelCase, but the variable `deletedOrder` uses camelCase as well. It should be consistent throughout the code. Consider renaming `deletedOrder` to `deletedorder` to follow the existing convention.